<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodesEditorDialogWindow</class>
 <widget class="QDialog" name="NodesEditorDialogWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактор состава ИД</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="hl_main">
     <item>
      <widget class="QTreeWidget" name="tw_nodes">
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="vl_buttons">
       <item>
        <widget class="QPushButton" name="btn_add_form">
         <property name="text">
          <string>Добавить форму</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_add_group">
         <property name="text">
          <string>Добавить группу</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_edit">
         <property name="text">
          <string>Изменить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_delete_item">
         <property name="text">
          <string>Удалить элемент</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="vert_spacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hl_saveclose">
     <item>
      <widget class="QPushButton" name="btn_save">
       <property name="text">
        <string>Сохранить</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons/resources/icons/save.svg</normaloff>:/icons/resources/icons/save.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_close">
       <property name="text">
        <string>Закрыть</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons/resources/icons/close.svg</normaloff>:/icons/resources/icons/close.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
